### ISSUE TITLE
Reduce shield speed

### ISSUE THREAD

**Issue by GRhin** [Sh0wdown/UnofficialCrusaderPatch#514]
I have been having lots of discussions around unit balancing, especially with regard to multiplayer,  and the one update that almost everybody agrees with is a speed reduction for shields.
Currently shields are faster than any unit except horse (including engineers without a shield) which makes no logical sense and means the shields cannot be countered. Even knights, while marginally faster, cannot catch up with them unless an entire map is crossed, catapults and traps are slow, inaccurate, and expensive so cant really break shields except when the player makes a big mistake.
So my request is to make shields travel at the same speed as crossbowmen and pikemen. This allows spears and cavalry to catch them, maces can also catch them if they can get into their final boost phase before the shields pull away, pikemen can still be used as defense to protect the shields from the relatively weak spears without slowing the shields further.
IN SUMMARY: making shields run at pike man speed will create a much more dynamic game as spears and maces, which are uncommon in multiplayer, have an important use, and the shields will have a cost to using them (speed) with horse archers, and makes alot more logical sense as well.

**Comment by Krarilotus** [comment]
Can you link to the orgininal discussions?
I would say this is a thing i would love to see for the next release, to appear in the unit-balance tab. Just not ticked by default tho, as its a huge change!

**Comment by ByBurton** [comment]
@Krarilotus A huge change? Yes. But who uses shields in a mobile way anyways? I mean besides in pvp.
The AI just puts them somewhere and it's good (or runs around with them all around the map).
Most people just put them somewhere, so they are basically stationary. In that case the change isn't that big.
They definitely are the most unbalanced unit right now, without a doubt. Their speed should match those of a monk imo.

**Comment by Krarilotus** [comment]
> @Krarilotus A huge change? Yes. But who uses shields in a mobile way anyways? I mean besides in pvp.
> The AI just puts them somewhere and it's good (or runs around with them all around the map).
> Most people just put them somewhere, so they are basically stationary. In that case the change isn't that big.
> They definitely are the most unbalanced unit right now, without a doubt. Their speed should match those of a monk imo.

Yes this is by far a way bigger balance change than swordsmans attack against walls, spearmans health or laddermans costs.
Its also the ONLY change so far that would really impact multiplayer. I have never seen a player just putting shields somewhere, they always used those when i saw it to protect their moving forces. Tho we might know different players and i can see your argument making sense. Its just that shields get abused so hard because of their speed, i definitly see this change as a must!

**Comment by ByBurton** [comment]
That is because you play with other pretty good players, and you play quick matches, for domination / competitively.
There are a lot of players that I know of that just play casually, focusing on building pretty pretty castles and stuff.
I for my part never use shields except really sparingly if I use fire throwers and battering rams.
And my friends never use them at all.

**Comment by Krarilotus** [comment]
Well, but for them the unit changes in the UCP are also not interesting at all. I would say this change is made for competitive players. Against AI or online, that doesnt matter. If you can abuse shields cuz of their speed against AI aswell, that gives AI such a huge disadvantage, that you coul argue, not to play with shields at all. And i dont like excluding stuff out of the game, just because its not balanced properly

**Comment by ByBurton** [comment]
I didn't say I want them out of the game. I just don't use them. All I wanted to say that the change is for the common folk not a huge a change.

**Comment by Krarilotus** [comment]
Ok agreed

**Comment by GRhin** [comment]
In the case of stationary shields they are not as OP as catapults and trebs can deal with them, however mobile shields are gone before rocks can arrive, cats are can be run down before follow up shots, and trebs, which can be put behind walls, are so slow to fire that trying to anticipate the shield location is incredibly difficult. Changing the shield speed wont change that too much, but it will allow other units to counter shields.

**Comment by Russianstory** [comment]
Yes, a good idea and idea. Shields cause inconvenience at their high speed.
We need to make them a little slow. But I still would have made spearmen anyway always running. Easy cheap unit for a quick attack or for cleansing from harmful slaves and slingers in your fortress.
-
Да хорошая идея и задумка. Щиты доставляют неудобства своей высокой скоростью.
Надо их сделать немного медленными. А вот копейщиков я всё-таки всё равно сделал бы всегда бегающими. Лёгкий дешёвый юнит для быстрой атаки или же для зачистки от вредных рабов и пращников в своей крепости.

**Comment by ByBurton** [comment]
@Russianstory I think they should always start running as soon as they are near an enemy (could function the same as the visibility check for assassins)

**Comment by Russianstory** [comment]
Yes, they have this function. BUT I do not understand why a lightweight almost naked unit in standard mode just walks? He must always run. It will also be if you want support for archers so that in close combat someone could cover them. It turns out the opposite - the shield walks but the speed is big ... and the spearman must run, but always walks .... except when you give the command "Aggressive position" or something like that

**Comment by Krarilotus** [comment]
@Russianstory can you make a different issue for this? i dont think we should discuss anything in here, and just get the shield speed done. Maybe even make it configurable -> monk speed, crossbowman speed, catapult speed, trebuchet speed *blink* :D

**Comment by LordHansCapon** [comment]
I think shields just work as other units regarding speed. When you select archers and pikemen for example, the archers move at the same speed as pikemen. The shields should do the same. If you select shields and other units, the selected units will move at the slowest unit's speed.

**Comment by Monsterfisch** [comment]
@LordHansCapon  that's already what they do but it can be easily bypassed by using shields in a separate control group to have them run their normal speed. the only saving grace is to make them slower by default and maybe have them only reduce a certain %chance to protect other units from taking damage

**Comment by Heroesflorian** [comment]
@LordHansCapon The main reason behind the proposal is that in competitive play, many players use lots of shields together with lots of horse archers as a very fast and near unkillable "death ball" that can dodge and outrun pretty much any melee unit, and shoot to death anything because the shields tank so well. 
Thus, reduce (maximum) movement speed of shields in order to make that strategy less powerful, as then horse archers will either adjust to the slower shield speed when moving, sacrificing part of their mobility for protection, or they will ride ahead without shields, losing their protection while moving.
